Podfile

```diff
@@ -9,6 +9,7 @@ target 'iOS-SDK' do
 	pod 'SDWebImage', '~> 4.0'
 	pod 'Alamofire', '~> 4.5'
 	pod 'Result', '~> 3.0.0'
-	pod 'SchedJoulesApiClient'
+    pod 'SchedJoulesApiClient', :git => 'https://github.com/schedjoules/swift-api-client.git', :branch => '58-subscriptions'
```

Comment:
Please open a PR on swift-api-client for that branch first. The subscription changes should be in master before we merge this one.

Reply:
PR is in the swift-api-client repo